packages/hardhat-core/sample-projects/typescript-viem/hardhat.config.ts

```diff
@@ -1,4 +1,5 @@
-import { HardhatUserConfig } from "hardhat/config";
+// @ts-check
```

Comment:
Hey @sambacha, can you walk me through the advantage of adding `// @ts-check` here?